rdt/transformers/numerical.py

```diff
@@ -299,6 +300,7 @@ def __init__(self, model_missing_values=None, learn_rounding_scheme=False,
         super().__init__(
             model_missing_values=model_missing_values,
             missing_value_generation=missing_value_generation,
+            missing_value_replacement='mean',  # Using this as default instead of random
```

Comment:
Why?

Reply:
Mainly because before we were using `mean` values as the only option for this transformer and @npatki suggested that this could stay the same as we would need to check how this propagates down the line to the synthesizer that uses this transformer.

Reply:
I'd remove the comment (or improve it) if that's the case, it doesn't explain to someone reading the code why you are doing it.